sphinx/tutorials/installation.rst

```diff
@@ -53,6 +53,15 @@ Option 2: Clone the library
    `MOxUnit <https://github.com/MOxUnit/MOxUnit>`__, and the tools needed
    for semidefinite programming.
 
+   Ignore the changes in the SDPT3 submodule, for example when compiling MEX files:
```

Comment:
I guess the user might not understand that he should do this (he might not be aware that mex files will be compiled for him in the SDPT3 submodule, and thus choose to ignore the suggestion rather than follow it). I'ld suggest "To ignore changes that will happen when the SDPT3 submodule is compiled, run the following command:"